### ISSUE TITLE
Cannot delete workspace :remote file operation failed

### ISSUE THREAD

**Issue by pakrym** [dotnet/dotnet-ci#1042]
We get a lot of Jenkins failures like https://ci3.dot.net/job/aspnet_IISIntegration/job/master/job/windows-Configuration_Release_prtest/75/console in https://github.com/aspnet/IISIntegration repo.

I suspect that it's caused by hanging w3wp or iisexpress processes that are not killed by default targets. Is there a way for us to kill processes before `WS-CLEANUP` runs or should I open PR to add them to common targets?

**Comment by mmitche** [comment]
This is the initial workspace cleanup, which means that a previous run still has processes running. AFAIK, this shouldn't happen because the previous run's processes should be killed by this time.

They are cleaned up by doing the following:
- Job has a cookie environment variable.  All processes started by the job get the cookie.
- After run, find all processes in the system with the cookie and kill.  This of course requires perms to kill those processes.

Could something you're doing be breaking that workflow?

**Comment by pakrym** [comment]
Yeah, we use full IIS and it's a service that runs from `SYSTEM` account.
